packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarUtils.tsx

```diff
@@ -6,27 +6,16 @@ interface DataToolbarContextProps {
   toggleIsExpanded: () => void;
   expandableContentRef: RefObject<HTMLDivElement>;
   expandableContentId: string;
+  chipGroupContentRef: RefObject<HTMLDivElement>;
+  updateShowClearFiltersButton: (showClearFiltersButton: boolean) => void;
```

Comment:
@nicolethoen looks like you missed removing a `updateShowClearFiltersButton` here